monitor/getShortEventStats.js

```diff
@@ -24,9 +29,19 @@ async function main(bridgeMode) {
       }
     }
   } else {
+    const unclaimedStats = {}
+    if (MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST || MONITOR_HOME_TO_FOREIGN_BLOCK_LIST) {
+      const unclaimedPool = homeToForeignRequests
+        .filter(eventWithoutReference(homeToForeignConfirmations))
+        .filter(unclaimedHomeToForeignRequests())
+
+      unclaimedStats.unclaimed = unclaimedPool.length
+      unclaimedStats.unclaimedDiff = Web3Utils.fromWei(BN.sum(...unclaimedPool.map(e => e.value)).toFixed())
```

Comment:
let's call it `unclaimedBalance` and reduce `balanceDiff` by this value. So, we will see: "ok, due to bug in oracle we have such balance diff where as we have unclaimedBalance for the operations that were not finalized by users"